crates/sui-cluster-test/src/faucet.rs

```diff
@@ -41,7 +43,22 @@ impl FaucetClientFactory {
                 .await
                 .unwrap();
 
-                Arc::new(LocalFaucetClient::new(simple_faucet))
+                let faucet = Arc::new(simple_faucet);
+                let faucet_batch = Arc::clone(&faucet);
```

Comment:
I think you can call this as a member function.

```suggestion
                let faucet_batch = faucet.clone();
```